File: app/src/main/java/ru/mai/app/data/repository/specification/StaticListContentSpecification.java
package ru.mai.app.data.repository.specification;

/**
 * Created by olegosipenko on 13.09.15.
 */
public class StaticListContentSpecification implements RequestSpecification<String> {
    private String item;

    public StaticListContentSpecification(String item) {
        this.item = item;
    }

    @Override
    public boolean specified(String s) {
        return s.equals(item);
    }

    @Override
    public String getItem() {
        return item;
    }
}

File: app/src/main/java/oleg/osipenko/mai/data/repository/specification/StaticListContentSpecification.kt
package oleg.osipenko.mai.data.repository.specification

import oleg.osipenko.mai.data.repository.specification.RequestSpecification

/**
 * Created by olegosipenko on 05.09.15.
 */
public class StaticListContentSpecification : RequestSpecification<String> {

    override fun specified(s: String): Boolean {
        return false
    }
}
